src/runUtils.ts

```diff
@@ -93,5 +94,11 @@ export function getHostOutputDir(): string {
 
 // This FILES_DIR is an env var coming from the user
 // JS_SANDBOX_OUTPUT_DIR is kept for retrocompatibility as this is the name of the old env var
```

Comment:
Given that you're removing the `JS_SANDBOX_OUTPUT_DIR` retrocompatibility, do you mind removing the comment as well?